pkg/yttlibrary/assert.go

```diff
@@ -36,3 +38,21 @@ func (b assertModule) Fail(thread *starlark.Thread, f *starlark.Builtin, args st
 
 	return starlark.None, fmt.Errorf("fail: %s", val)
 }
+
+func (b assertModule) TryTo(thread *starlark.Thread, f *starlark.Builtin, args starlark.Tuple, kwargs []starlark.Tuple) (starlark.Value, error) {
+	if args.Len() != 1 {
+		return starlark.None, fmt.Errorf("expected exactly one argument")
+	}
+
+	lambda := args.Index(0)
+	if _, ok := lambda.(starlark.Callable); !ok {
+		return starlark.None, errors.New("expected argument to be a callable function or lambda")
```

Comment:
lets change this a bit to be more consistent with [0]:

```
Expected argument to be a function, but was %T
```

https://github.com/vmware-tanzu/carvel-ytt/blob/develop/pkg/yttlibrary/overlay/assert_annotation.go#L113-L114

Reply:
nit: lets also just use fmt.Errorf. no need to import errors after that.